doc/development/create-a-model-pt.md

```diff
@@ -137,6 +137,15 @@ class SomeAtomicModel(BaseAtomicModel, torch.nn.Module):
         pass
 ```
 
+### Floating-point precision
+
+When creating a new component, the floating-point precision should obey the [Floating-point precision of the model](../model/precision.md) section.
+In implementation, the component should
+
+- store parameters in the component precision, except those for input and output normalization;
```

Comment:
do we need to store the input normalization in interface precision? current implementation does not:
https://github.com/wanghan-iapcm/deepmd-kit/blob/0c6b42a51c809141ed757639ebd82c781c9eda0e/deepmd/pt/model/descriptor/se_a.py#L423-L424
the inputs are converted to internal precision anyway and we do not expect large bias/std for inputs.

Reply:
In TF, it does:

https://github.com/deepmodeling/deepmd-kit/blob/02580c2fd9e1f35b7fb9b3e6320a67deb553f163/deepmd/tf/descriptor/se_a.py#L627-L641

Reply:
these parameters will any way be cast to internal precision, which is potentially lower than global precision.

Reply:
I don't think it is cast in TF, so it's questionable what is the best way

https://github.com/deepmodeling/deepmd-kit/blob/02580c2fd9e1f35b7fb9b3e6320a67deb553f163/deepmd/tf/descriptor/se_a.py#L651-L664

https://github.com/deepmodeling/deepmd-kit/blob/02580c2fd9e1f35b7fb9b3e6320a67deb553f163/deepmd/tf/fit/ener.py#L549-L579
https://github.com/deepmodeling/deepmd-kit/blob/02580c2fd9e1f35b7fb9b3e6320a67deb553f163/deepmd/tf/fit/ener.py#L598-L602

In PT, it's confusing that fparam and aparam are not cast, although the normalization parameters are stored in the fitting precision

https://github.com/deepmodeling/deepmd-kit/blob/02580c2fd9e1f35b7fb9b3e6320a67deb553f163/deepmd/pt/model/task/fitting.py#L419-L442

Reply:
https://github.com/deepmodeling/deepmd-kit/pull/4240/commits/5958e8f135b787e61ec4754080068025fa5fadfd